app/views/documents/forbidden.html.erb

```diff
@@ -0,0 +1,5 @@
+<% content_for :title, "Forbidden" %>
+
+<div class="govuk-body" data-gtm="access-limit-forbidden" data-gtm-visibility-tracking="true">
```

Comment:
Do these data attributes have any effect in Collections Publisher?

Reply:
Removed.